UnitySDK/Assets/ML-Agents/Scripts/Policy/BehaviorParameters.cs

```diff
@@ -11,16 +11,28 @@ namespace MLAgents
     public class BehaviorParameters : MonoBehaviour
     {
 
+        [Serializable]
+        private enum BehaviorType
+        {
+            DEFAULT,
```

Comment:
nit: I think these should be CamelCase according to the C# coding standard (https://ono.unity3d.com/unity-extra/unity-meta/files/@/ReferenceSource/CSharp/Assets/CSharpReference.cs#L133 even though it doesn't actually say that, just what it shouldn't be). I don't care much as long as it looks good in the drop-down.